pydaikin/daikin_brp084.py

```diff
@@ -0,0 +1,614 @@
+                        "/dsiot/edge/adr_0100.dgc_status",
+                    )
+                )
+
+    def _handle_swing_setting(self, settings, requests):
+        """Handle swing-related settings."""
+        if (
+            'f_dir' in settings
+            and self.values['mode'] in self.HVAC_MODE_TO_SWING_ATTR_NAMES
+        ):
+            vertical_attr_name, horizontal_attr_name = (
+                self.HVAC_MODE_TO_SWING_ATTR_NAMES[self.values['mode']]
+            )
+
+            if settings['f_dir'] in ('off', 'horizontal'):
+                # Turn off vertical swing
+                requests.append(
+                    DaikinAttribute(
+                        vertical_attr_name,
+                        self.TURN_OFF_SWING_AXIS,
+                        ["e_1002", "e_3001"],
+                        "/dsiot/edge/adr_0100.dgc_status",
+                    )
+                )
+            else:
+                # Turn on vertical swing
+                requests.append(
+                    DaikinAttribute(
+                        vertical_attr_name,
+                        self.TURN_ON_SWING_AXIS,
+                        ["e_1002", "e_3001"],
+                        "/dsiot/edge/adr_0100.dgc_status",
+                    )
+                )
+
+            if settings['f_dir'] in ('off', 'vertical'):
+                # Turn off horizontal swing
+                requests.append(
+                    DaikinAttribute(
+                        horizontal_attr_name,
+                        self.TURN_OFF_SWING_AXIS,
+                        ["e_1002", "e_3001"],
+                        "/dsiot/edge/adr_0100.dgc_status",
+                    )
+                )
+            else:
+                # Turn on horizontal swing
+                requests.append(
+                    DaikinAttribute(
+                        horizontal_attr_name,
+                        self.TURN_ON_SWING_AXIS,
+                        ["e_1002", "e_3001"],
+                        "/dsiot/edge/adr_0100.dgc_status",
+                    )
+                )
+
+    async def set(self, settings):
+        """Set settings on Daikin device."""
+        await self._update_settings(settings)
+        requests = []
+
+        # Handle different types of settings
+        self._handle_power_setting(settings, requests)
+        self._handle_temperature_setting(settings, requests)
+        self._handle_fan_setting(settings, requests)
+        self._handle_swing_setting(settings, requests)
+
+        if requests:
+            request_payload = DaikinRequest(requests).serialize()
+            _LOGGER.debug("Sending request: %s", request_payload)
+            response = await self._get_resource("", params=request_payload)
+            _LOGGER.debug("Response: %s", response)
+
+            # Update status after setting
+            await self.update_status()
+
+
+# pylint: disable=unused-argument
+async def set_holiday(self, mode):
```

Comment:
Make sure this returns `False` instead.

https://github.com/fredrike/pydaikin/blob/master/pydaikin/daikin_base.py#L310